src/PyScriptEC/PyScriptEncConverter.cs

```diff
@@ -129,13 +129,16 @@ bool                bForward
                 // We could use UTF-8 here, but wide data works just fine.
                 // the windows version definitely needs UTF16, but for some reason _MSC_VER is not defined on windows (as I think Jim is expecting)
                 //  so I'll use the opposite logic of 'if not mono'...
-#if !__MonoCS__ // _MSC_VER
-                Util.DebugWriteLine(this, "eInFormEngine UTF16");
-                eInFormEngine = EncodingForm.UTF16;
-#else
-                Util.DebugWriteLine(this, "eInFormEngine UTF32");
-                eInFormEngine = EncodingForm.UTF32;
-#endif
+                if (!Util.IsUnix) // _MSC_VER

```

Comment:
It would be easier to if you'd switch the two cases and put `Util.IsUnix` first.

Reply:
Why is it easier? Done.

Reply:
It's easy to overlook the `!`, and the two blocks don't make it obvious which one is for which platform. 
Usually I think it's recommended practice if you have both `if` and `else` that you order the blocks in such a way that you don't need the negation.

Reply:
That makes sense, thanks.